src/core/icons/fileIcons.ts

```diff
@@ -1228,6 +1228,8 @@ export const fileIcons: FileIcons = {
       name: 'nodejs',
       fileNames: [
         'package.json',
+        'package.yaml',
+        'package.json5',
```

Comment:
I'm not sure about the `package.yaml` file because I don't see it in all JS ecosystems, it's only present in **pnpm**
But I think we can add `package.json5`

If you agree:

```suggestion
        'package.json5',
```

Reply:
Even JSON5 is not supported:
https://github.com/npm/feedback/discussions/56#discussioncomment-8610115

Reply:
> Even JSON5 is not supported: [npm/feedback#56 (comment)](https://github.com/npm/feedback/discussions/56#discussioncomment-8610115)

pity

Reply:
Perhaps we need to close the PR